.github/workflows/ci-dgraph-code-coverage.yml

```diff
@@ -0,0 +1,75 @@
+name: ci-dgraph-code-coverage
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - main
+  schedule:
+    - cron: "0 22 * * *"
+jobs:
+  dgraph-code-coverage:
+    # TODO: uncomment this after release work
+    # if: github.event.pull_request.draft == false
```

Comment:
could you uncomment this out for this CI job? I feel like we should run this only on non-draft always! we have commented this out temporarily for other jobs ... but for this job I think it should always stay uncommented.